test/flint/padic-test.jl

```diff
@@ -25,6 +25,8 @@
 
    @test isa(fmpz(1)//7^2 + fmpz(2)//7 + 3 + 4*7 + O(R, 7^2), padic)
 
+   @test precision( R(fmpq(2//3)^100) ) == precision( R(fmpq(2//3))^100 )
```

Comment:
Our convention is to not put spaces immediately inside parentheses. Of course this convention only applies to AbstractAlgebra and Nemo as it will be too hard to enforce that kind of uniformity elsewhere.

Reply:
Got it. Thanks!